### ISSUE TITLE
status: InsertEventLog can duplicate console log output

### ISSUE THREAD

**Issue by mrtracy** [cockroachdb/cockroach#9187]
Method `EventLogger.InsertEventRecord()` inserts an event into our SQL event table.  At the beginning of the method, it also logs information to the console about the event:

``` go
// InsertEventRecord inserts a single event into the event log as part of the
// provided transaction.
func (ev EventLogger) InsertEventRecord(txn *client.Txn, eventType EventLogType, targetID, reportingID int32, info interface{}) error {
    // Record event record insertion in local log output.
    log.Infof(txn.Context, "Event: %q, target: %d, info: %+v",
        eventType,
        targetID,
        info)
```

However, InsertEventRecord is always run in the context of a client.Txn. If that transaction is automatically retried, the console log message can appear multiple times (even though only one event has been recorded to the SQL table).

**Comment by dianasaur323** [comment]
@mrtracy was this fixed after the work we did on the event log?

**Comment by mrtracy** [comment]
Yes, it was fixed by #10251. Closing.
